source/manual_guidelines.rst

```diff
@@ -1,272 +0,0 @@
-.. include:: /shortcuts.rstext
-
-Guidelines for Mixxx Manual writers
```

Comment:
Where else is this info?

Reply:
We could put it in the wiki or a Readme file

Reply:
Maybe a readme file is not accessible ..

Reply:
It's certainly more accessible than a file that is hidden in the source directory and isn't even part of any toctree. I think GitHub supports `rst` files, I'll just move it into the repo root and link it from the readme.

Reply:
Done.